applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_broadcast.c

```diff
@@ -210,11 +215,6 @@ static void pa_synced_cb(struct bt_le_per_adv_sync *sync,
 	int ret;
 	struct bt_mgmt_msg msg;
 
-	if (sync != pa_sync) {
```

Comment:
Why can this be removed?

Reply:
This is because the way we got pa_sync is not only from `periodic_adv_sync()` which from normal scan.
We can also get PAST from the broadcast assistant, which triggered `pa_synced_cb()` directly.